.github/workflows/ci-unix-shared-installed.yml

```diff
@@ -35,11 +35,19 @@ jobs:
       working-directory: ./ext
       run: sed -i'' -e 's/-DBUILD_SHARED_LIBS=OFF/-DBUILD_SHARED_LIBS=ON/' *.cmd
     - name: Add recent repository for libyuv-dev
+      if: runner.os == 'Linux'
       run: echo "deb http://azure.archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
     - name: Update apt
+      if: runner.os == 'Linux'
       run: sudo apt-get update
-    - name: Install dependencies
+    - name: Install dependencies (Linux)
+      if: runner.os == 'Linux'
       run: sudo apt-get install libaom-dev libdav1d-dev libyuv-dev
+    - name: Install dependencies (macOS)
+      if: runner.os == 'macOS'
+      run: brew install aom dav1d
+      # TODO(yguyon): Install libyuv
```

Comment:
Nit: It looks nicer to put the TODO comment before the `brew install` command. Or is it because we will need to install libyuv using a different package management system on macOS?

Reply:
Yes, `libyuv` is not available with `brew`. I found it on [MacPorts](https://ports.macports.org/port/libyuv/), but it is not available on the GitHub macOS VM by default, and `installer <pkg>` did not work ("installer not found").